processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -71,6 +71,15 @@ func (d *Detector) Detect(ctx context.Context) (resource pcommon.Resource, schem
 		return pcommon.NewResource(), "", fmt.Errorf("failed getting host architecture: %w", err)
 	}
 
+	var hostIPAttribute []any
+	hostIPs, errIPs := d.provider.HostIPs()
+	if errIPs != nil {
+		return pcommon.NewResource(), "", fmt.Errorf("failed getting host IP addresses: %w", errIPs)
+	}
+	for _, ip := range hostIPs {
+		hostIPAttribute = append(hostIPAttribute, ip.String())
+	}
```

Comment:
I'd be good to wrap it in `d.cfg.ResourceAttributes.HostIP.Enabled` so errors are not returned if the attribute is disabled

Reply:
Makes sense, done